stable/jenkins/values.yaml

```diff
@@ -81,3 +82,10 @@ Persistence:
 #  - mountPath: /var/nothing
 #    name: nothing
 #    readOnly: true
+
+NetworkPolicy:
+  # Enable creation of NetworkPolicy resources.
+  Enabled: true
```

Comment:
Should we leave this as false for the moment and give people the option to enable?